lib/checks/label/label-content-name-mismatch.js

```diff
@@ -0,0 +1,25 @@
+const {
+	text: {
+		accessibleText,
+		visible // get visible text
+	}
+} = axe.commons;
+
+/**
+ * Note:
+ * `label-content-name-mismatch-matches` ignore `node`
+ * if there is no `accessibleText` or `textContent`
+ */
+const accText = accessibleText(node).toLowerCase();
+const visibleTextContent = visible(node).toLowerCase();
+
+/**
+ * if `textContent` is not part of `accessibleText` -> fail
+ */
+
+if (!accText.includes(visibleTextContent)) {
+	return false;
```

Comment:
We need to do something about `<button aria-label="close">x</button>`. It might be sufficient to put elements with text content of a single char into needs review?